### ISSUE TITLE
[v1.164.0] missing images in payments

### ISSUE THREAD

**Issue by jrgm** [mozilla/fxa#4669]
I happened to have the browser console open and noticed two 404 on images from payments in stage with v1.164.0. (They may have been missing before this release, mind you).

Can confirm from command line too. Note: there is a firefox-logo.svg in the git repo, but there is no `icon-check-white.svg` in the git repo.

```
$ curl -s -D - 'https://payments-stage.fxa.nonprod.cloudops.mozgcp.net/images/icon-check-white.svg' | head -3
HTTP/2 404
server: openresty
date: Thu, 26 Mar 2020 21:39:28 GMT
$ curl -s -D - 'https://payments-stage.fxa.nonprod.cloudops.mozgcp.net/images/firefox-logo.svg' | head -3
HTTP/2 404
server: openresty
date: Thu, 26 Mar 2020 21:39:42 GMT
```

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1424)
┆Issue Number: FXA-1424

**Comment by lmorchard** [comment]
Hmm, yeah, neither of those images are here: https://github.com/mozilla/fxa/tree/master/packages/fxa-payments-server/public/images

Looks like icon-check-white is referenced in a content-server style - if we're using that on payments-server, we'll need to copy that image over: https://github.com/mozilla/fxa/search?q=%2Fimages%2Ficon-check-white.svg&unscoped_q=%2Fimages%2Ficon-check-white.svg

Not sure where the firefox-logo.svg is missing, though

**Comment by data-sync-user** [comment]
➤ John Morrison commented:

I just went into stage, signed up for a new account, then went to 123done-stage.dev.lcip.org, signed in, then went through the steps to get a 123done PRO subscription.

I did it again now. I now longer see a 404 on firefox-logo.svg, but I do see the 404 on icon-check-white.svg when loading the credit card form, and also, after subscribing, when clicking on Cancel to start a cancellation.
